### DISCUSSION TITLE
Incorporate Photometer as a sensor into Maui.Essentials

### DISCUSSION THREAD

**Discussion by Mrnikbobjeff** [dotnet/maui#22631] [Ideas]
Referencing https://github.com/xamarin/Essentials/issues/407
With iOS 14 the Ambient Light sensor has become available for iOS. At the time the ticket was discussed this was not available.
Since i still have a working implementation for the other platforms in one of my old Xamarin.Essentials branches I thought I might bring this up again.
For the iOS Docs, see [here](https://developer.apple.com/documentation/sensorkit/srsensor/3377673-ambientlightsensor).

**Comment by Mrnikbobjeff** [comment]
Tagging @mattleibow as he participated in the old repo

**Reply by mattleibow** [reply]
Maybe update the description with the proposed APIs and what implementations you have. The Essentials issue mentioned that not all platforms have this, however I am not sure that is a factor that should stop us. WinUI/Windows has less sensors and macOS basically has none.

If this helps iOS and Android, then this is great.

If this is something that the community wants and the code is there, then I don't see why we _won't_ accept it, but it will be up to the rest of the team to decide.

You mentioned an old branch in Essentials, where/which one is that?

**Reply by Mrnikbobjeff** [reply]
https://github.com/Mrnikbobjeff/Essentials/tree/Luminosity is the branch, it contains Android, UWP and the outdated iOS code only though.
If you might take a look at my cancelled PR's in the old Essentials Repo you will find some more features which might be considered in scope now. Audio Recording and State tracking come to mind.
